### DISCUSSION TITLE
ch8/ch8-09

### DISCUSSION THREAD

**Discussion by giscus** [gopl-zh/gopl-zh.github.com#169] [General]
# ch8/ch8-09

<The Go Programming Language>中文版

https://gopl-zh.github.io/ch8/ch8-09.html

**Comment by zhangbest5** [comment]
在结束之前我们需要把fileSizes channel中的内容“排”空? fileSizes 哪来的？这突然和上节的例子没关系了，我得找个全代码看看这是干了些啥？

**Reply by Etherealss** [reply]
上一节的例子中不就一直在用fileSizes吗

**Comment by zhangbest5** [comment]
？
我记得之前提了的，怎么没了？
bug反馈，与导航目录不符，目录里是8.8，这写的8.9，导致下面8.9变成了8.10，也与目录不符

**Comment by chai2010** [comment]
如果发现问题可以看看能否顺手提个PR，感谢

**Reply by zhangbest5** [reply]
pr是项目评审(Project Review)英文的缩写？主要不会啊，所以在下面评论反馈~感谢作者翻译

**Reply by zhangbest5** [reply]
我看github上，有ch8-08.md 8.8. 示例: 并发的目录遍历，9和10也有，是把8丢了，造成了目录顺序不对应

**Reply by chai2010** [reply]
买本github实操的入门书，刚好实战一把。PR 是 pull request 的缩写，这是github平台的黑话
